### PR TITLE
added handlebars templates

### PR DESCRIPTION
**Pull request by pattt**
Let's add some templating :octocat:

### CODE REVIEW

**Comment by stargazer33** [comment]
pls rename `_includes/themes/restart/content.html` to `_includes/themes/restart/page-jobad-jobdetails-handlebars.html`

File: _includes/themes/restart/default.html [hunk 1]
```diff
@@ -112,4 +113,5 @@
   {% endif %}
   {% include JB/analytics %}
   </body>
+{% include themes/restart/content.html %}
```
Comment (stargazer33) [inline _includes/themes/restart/default.html]: Move this `include` inside the if statement which begins at line 109 ( I mean this if: `{% if thisnodes[0].datafile %}` )

This way the template will be added only to the "job listing" pages and not to other pages like index.html

File: assets/themes/restart/js/render_jobad_list_filter.js [hunk 2]
```diff
@@ -383,32 +383,10 @@ function tableLoad(isFirstLoad) {
...
-            html.push('</div>');
-            html.push('</div>'); // div class="col-xs-12 col-sm-12 col-md-12"
-        html.push('<span class="detailFooter">&nbsp;</span>');
-
-    html.push('</div>'); //highlight
-    html.push('</div></div>'); //detailContent row
-    html.push('</td></div>'); //detail-view
-    return html.join('');
+    var source   = document.getElementById("detailFormatter").innerHTML,
+        template = Handlebars.compile(source),
```
Comment (stargazer33) [inline assets/themes/restart/js/render_jobad_list_filter.js]: I think the lines 386 and 387 should be executed only once, when the page loaded/initialized.
Is it correct?
If the answer is yes than pls move these 2 lines info function `$(document).ready(function () { `

File: _includes/themes/restart/default.html (hunk 1 above)
Comment (pattt) [reply _includes/themes/restart/default.html]: ok

File: assets/themes/restart/js/render_jobad_list_filter.js (hunk 2 above)
Comment (pattt) [reply assets/themes/restart/js/render_jobad_list_filter.js]: true

File: assets/themes/restart/js/render_jobad_list_filter.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ var isLunarIndexLoaded = false;
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
 var mapIdToJob = new Map();
 var json_data_original = [];
+var template = null;
```
Comment (stargazer33) [inline assets/themes/restart/js/render_jobad_list_filter.js]: Pls rename `template` to something like `detailFormatterTemplate`.
After this - LGTM

Comment (pattt) [reply assets/themes/restart/js/render_jobad_list_filter.js]: ok

**Pull request merged**